src/library-authoring/course-import/CourseImportPage.jsx

```diff
@@ -179,6 +179,19 @@ export const CourseImportListFilter = ({
     });
   };
 
+  const renderOption = option => {
+    if (option.group) {
+      return (
+        <optgroup label={option.label}>
+          {option.group.map(renderOption)}
+        </optgroup>
+      );
+    }
+    return (
+      <option value={option.value} key={option.value}>{option.label}</option>
+    );
+  };
```

Comment:
i noticed there is an identical function in `LibraryListPage.jsx`, is there any way this could be put somewhere that allows it to be used in both files?

Reply:
`renderOption` isn't used in `LibraryListPage.jsx` anymore so I have removed it. Don't think we should make it a util now